packages/components/src/__rc__/Select/Select.tsx

```diff
@@ -173,13 +177,53 @@ export const Select = <Option extends SelectOption = SelectOption>({
           trigger(selectToggleProps, selectToggleProps.ref)
         )}
         {state.isOpen && (
-          <Popover id={popoverId} portalContainer={portalContainer} refs={refs}>
+          <RacPopover
+            className={styles.popover}
+            isOpen={state.isOpen}
+            defaultOpen={defaultOpen}
+            onOpenChange={state.setOpen}
+            triggerRef={selectToggleProps.ref}
+            // aria-labelledby={trigger.props.id}
+            offset={15}
+            // isNonModal
+            // shouldUpdatePosition
+            placement="bottom start"
+            maxHeight={352}
+            containerPadding={12}
+            scrollRef={popoverContentsRef}
+            UNSTABLE_portalContainer={portalContainer}
+            // id={popoverId}
+            // portalContainer={portalContainer}
+          >
+            <Dialog id={popoverId} aria-labelledby={triggerProps['aria-labelledby']}>
+              <div ref={popoverContentsRef}>
+                <SelectProvider<Option> state={state}>
+                  <SelectPopoverContents menuProps={menuProps}>{children}</SelectPopoverContents>
+                </SelectProvider>
+              </div>
+            </Dialog>
+          </RacPopover>
+        )}
+      </div>
+      {/* {state.isOpen && (
```

Comment:
This is an alternative that I have briefly looked at - which would be to handle this like we do the MultiSelect